discovery-provider/src/queries/get_l2_cnode.py

```diff
@@ -0,0 +1,20 @@
+from src import exceptions
+from src.models import L2ContentNode
+from src.utils import helpers
+from src.utils.db_session import get_db_read_replica
+from src.queries.query_helpers import paginate_query
+
+def get_l2_cnodes():
```

Comment:
are filters necessary here?

Reply:
i kind of think no? as in querying for a subset etc. pagination should likely cover the requirements